viam-cartographer/scripts/setup_cartographer_macos.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
+which brew && export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:`brew --prefix openssl@3`/lib/pkgconfig"
```

Comment:
@Otterverse, @kkufieta said this did not work for her when she ran `make clean; make setup; make buf; make build`. Did I misunderstand your suggestion?

Reply:
I'm so sorry, I sent the above comment the other day, and didn't realize it'd been left in "pending" for my review!